test/plots/rectanglePlotTests.ts

```diff
@@ -2,9 +2,9 @@
 
 describe("Plots", () => {
   describe("RectanglePlot", () => {
-    describe("Basic usage", () => {
-      it("has correct rectangle heights and positions (renders correctly)", () => {
-        let data = [
+    describe("drawing", () => {
```

Comment:
Also, I notice that it says "drawing" here, but "rendering" in #2912. We should be consistent.